pallets/runtime/mainnet/src/runtime.rs

```diff
@@ -0,0 +1,338 @@
+            type LimitOrigin = polymesh_primitives::EnsureRoot;
+            // Can manage its own addition, deletion, and swapping of membership...
+            type AddOrigin = VMO<pallet_committee::$instance>;
+            type RemoveOrigin = Self::AddOrigin;
+            type SwapOrigin = Self::AddOrigin;
+            // ...but it cannot reset its own membership; GC needs to do that.
+            type ResetOrigin = VMO<GovernanceCommittee>;
+            type MembershipInitialized = $committee;
+            type MembershipChanged = $committee;
+            type WeightInfo = polymesh_weights::pallet_group::WeightInfo;
+        }
+    };
+}
+
+committee_config!(TechnicalCommittee, Instance3);
+committee_config!(UpgradeCommittee, Instance4);
+
+impl pallet_pips::Trait for Runtime {
+    type Currency = Balances;
+    type VotingMajorityOrigin = VMO<GovernanceCommittee>;
+    type GovernanceCommittee = PolymeshCommittee;
+    type TechnicalCommitteeVMO = VMO<pallet_committee::Instance3>;
+    type UpgradeCommitteeVMO = VMO<pallet_committee::Instance4>;
+    type Event = Event;
+    type WeightInfo = polymesh_weights::pallet_pips::WeightInfo;
+    type Scheduler = Scheduler;
+}
+
+/// CddProviders instance of group
+impl pallet_group::Trait<pallet_group::Instance2> for Runtime {
+    type Event = Event;
+    type LimitOrigin = polymesh_primitives::EnsureRoot;
+    type AddOrigin = polymesh_primitives::EnsureRoot;
+    type RemoveOrigin = polymesh_primitives::EnsureRoot;
+    type SwapOrigin = polymesh_primitives::EnsureRoot;
+    type ResetOrigin = polymesh_primitives::EnsureRoot;
+    type MembershipInitialized = Identity;
+    type MembershipChanged = Identity;
+    type WeightInfo = polymesh_weights::pallet_group::WeightInfo;
+}
+
+construct_runtime!(
+    pub enum Runtime where
+        Block = Block,
+        NodeBlock = polymesh_primitives::Block,
+        UncheckedExtrinsic = UncheckedExtrinsic
+    {
+        System: frame_system::{Module, Call, Config, Storage, Event<T>} = 0,
+        Babe: pallet_babe::{Module, Call, Storage, Config, Inherent, ValidateUnsigned} = 1,
+        Timestamp: pallet_timestamp::{Module, Call, Storage, Inherent} = 2,
+        Indices: pallet_indices::{Module, Call, Storage, Config<T>, Event<T>} = 3,
+
+        // Balance: Genesis config dependencies: System.
+        Balances: pallet_balances::{Module, Call, Storage, Config<T>, Event<T>} = 4,
+
+        // TransactionPayment: Genesis config dependencies: Balance.
+        TransactionPayment: pallet_transaction_payment::{Module, Storage} = 5,
+
+        // Identity: Genesis config deps: Timestamp.
+        Identity: pallet_identity::{Module, Call, Storage, Event<T>, Config<T>} = 6,
+        Authorship: pallet_authorship::{Module, Call, Storage, Inherent} = 7,
+
+        // CddServiceProviders: Genesis config deps: Identity
+        CddServiceProviders: pallet_group::<Instance2>::{Module, Call, Storage, Event<T>, Config<T>} = 38,
```

Comment:
For mainnet we can sequential numbering, I don't see a requirement of having an irregular sequence

Reply:
Done in https://github.com/PolymathNetwork/Polymesh/pull/925.